de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/AbstractIncomingReferencePointNegotiation.java

```diff
@@ -39,21 +39,22 @@
 // MAJOR TODO refactor this class !!!
 
 /**
- * Handles incoming ProjectNegotiations except for the actual file transfer.
+ * Handles incoming ReferencePointNegotiations except for the actual file
+ * transfer.
  * 
  * Concrete implementations need to provide an implementation to exchange the
  * calculated differences. This class only provides the initial setup and
  * calculation.
  */
-public abstract class AbstractIncomingProjectNegotiation extends
-    ProjectNegotiation {
+public abstract class AbstractIncomingReferencePointNegotiation extends
+    ReferencePointNegotiation {
 
     private static final Logger LOG = Logger
-        .getLogger(AbstractIncomingProjectNegotiation.class);
+        .getLogger(AbstractIncomingReferencePointNegotiation.class);
 
     private static int MONITOR_WORK_SCALE = 1000;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename this field "MONITOR_WORK_SCALE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. ([squid:S3008](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS3008))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=259&issues=AWZfKMe_DZywvPe4LUfX&open=AWZfKMe_DZywvPe4LUfX)

[ISSUE_KEY]: AWZfKMe_DZywvPe4LUfX